docs/concepts/controllers.md

```diff
@@ -0,0 +1,110 @@
+# Controllers
+
+etcd-druid is an operator to manage etcd clusters, and follows the [`Operator`](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/) pattern for Kubernetes.
+It makes use of the [Kubebuilder](https://github.com/kubernetes-sigs/kubebuilder) framework which makes it quite easy to define Custom Resources (CRs) such as `Etcd`s and `EtcdCopyBackupTask`s through [*Custom Resource Definitions*](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) (CRDs), and define controllers for these CRDs.
+etcd-druid uses Kubebuilder to define the `Etcd` CR and its correspdonding controllers.
```

Comment:
```suggestion
etcd-druid uses [Kubebuilder](https://book.kubebuilder.io/) to define the `Etcd` CR and its correspdonding controllers.
```

Reply:
`Kubebuilder` is linked to in **line 4** already (linked to its GitHub repository). Is linking to its book in **line 5** necessary @ishan16696?

Reply:
ok, then it's fine. thanks